recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy
+    - numpy x.x
```

Comment:
@ocefpaf, uhm good question. I saw that continuum pinned numpy in their channel and thought that this should be done generally. Is numpy only pinned if we build against it?